packages/components/Callout/macos/CalloutWindow.swift

```diff
@@ -26,11 +19,13 @@ class CalloutWindow: NSWindow {
 		backgroundColor = .clear
 		isMovable = false
 
-		NotificationCenter.default.addObserver(self, selector: #selector(appDidChangeActive(notification:)), name: NSApplication.didResignActiveNotification, object: nil)
+		// Dismiss the Callout if the window is no longer active.
+		NotificationCenter.default.addObserver(self, selector: #selector(dismissCallout), name: NSApplication.didResignActiveNotification, object: nil)
```

Comment:
where are you removing the observer?

Reply:
Do I need to? I thought they remove themselves when the containing object (window) is destroyed?

Reply:
you are right. i thought you were usingblock but you are not.